src/components/detailedLanding.js

```diff
@@ -93,10 +96,28 @@ const DetailedLanding = ({ userEntered, setUserEntered }) => {
     })();
   }, []);
   useEffect(() => {
-    (async () => {
-      const { data } = await axios.get(
+    function success(position) {
+      setWeatherAddress(
```

Comment:
Suggestion - there's no need of using state variable for this since this api link is not getting displayed on the screen. You can use normal variables for the assignment